### ISSUE TITLE
eqlogparser runs minimized w error trigger database not available. in use by another EQLogParser?

### ISSUE THREAD

**Issue by vanjm** [kauffman12/EQLogParser#169]
I had installed EQLogParser originally as EQLogParser-2.2.7.msi, then it looks like it downloaded and updated with EQLogParser-install-2.2.17.exe.  Now it tried to update to EQLogParser-install-2.2.19.exe and when I run it it stays minimized but shows 
![eqlogparsererror](https://github.com/kauffman12/EQLogParser/assets/170149383/8d6b704c-3778-4984-96ea-7905e27f5e1b)

**Comment by kauffman12** [comment]
That generally means two parsers are running as only one can access the
triggers DB at a time. Check your process list for a 2nd.

**Comment by vanjm** [comment]
Thank you!  Although I was trying to only get one parser running,  I think the real problem was that the display resolution for the windows 11 screen changed, and the os would not let the program launch in anything other than minimized.  Once I upped the resolution, the program loads normally.

**Comment by kauffman12** [comment]
Do you think the Windows 11 resolution was lowered? So the parser was trying to display too big? Maybe I can put in some checks for that.

**Comment by vanjm** [comment]
The resolution was lowered to 1600x900 instead of 1920x1080.  Seems more like a windows 11 error than an error in your program. And thank you for such a nice program.

**Comment by kauffman12** [comment]
Interesting. Thanks!

**Comment by vanjm** [comment]
Yes it does seem weird that there would be such a bug in windows 11. I suppose it could be that this isn't a super standard laptop because it is a touchscreen and the max resolution might be different?  But yes the screen resolution was also giving me issues with launching Everquest.  EQ was also going straight to minimized and had issues going to full screen from minimized.

**Comment by kauffman12** [comment]
This should be fixed in 2.2.20.  It tries to check if the parser is visible in any screen and if not it moves it to the primary.
